mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -4953,7 +5416,7 @@
 													"        //Get the first record",
 													"        let firstRecord = response.data[0];",
 													"        let secondRecord = response.data[1];",
-													"        //Test that type is packages",
+													"        //Test that type is providers",
 													"        pm.test('type is providers', function() {",
```

Comment:
same comment as above - are we checking for providers/packages?

Reply:
Thanks. Already fixed.